### PR TITLE
(#1207)Added link to map address

### LINKED ISSUE #1207: Open Source Map Link to be added to the Address
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
No link

**Expected behavior:**
Should be there

**Screenshots of the issue:**
screenshot 2018-11-14 at 9 26 34 pm
We can add a link to this address

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #1207 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

Added open street map link to address, check contact.component.html for more details.
https://pr-1237-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by saarthakchats** [comment]
I dont understand why Travis is failing

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h2 class="bold">Contact Us</h2>
     <div class="row">
       <div class="col-lg-12 col-sm-12 sub-details">
         <h5 class="bold">Susper</h5>
-        <p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br> Phone +84 (0) 907 65 29 27<br><a href="mailto: support@susper.net"> Email: support@susper.net</a><br>Board
+        <a href="https://www.openstreetmap.org/search?query=93%20Mau%20Than%20Street%2C%20Can%20Tho#map=19/10.03117/105.77529"><p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br></a> Phone +84 (0) 907 65 29 27<br><a href="mailto: support@susper.net"> Email: support@susper.net</a><br>Board
```
Comment (pythongiant) [inline src/app/contact/contact.component.html]: Please properly close your tags. that should fix the travis build

Comment (Ryan10145) [inline src/app/contact/contact.component.html]: @saarthakchats You start an <a> tag and then inside, you start a `<p>` tag. However, you then close the `<a>` tag before you close the `<p>` tag. This is causing the issues. I think you can fix the issue by starting the `<p>` tag before the `<a>` tag.

**Comment by saarthakchats** [comment]
@praveenojha33 Please merge

**Comment by saarthakchats** [comment]
@simsausaurabh please review and merge, thanks

**Pull request merged**